aca_preview/__init__.py

```diff
@@ -1 +1,8 @@
 __version__ = '4.0'
+
+def test(*args, **kwargs):
+    '''
```

Comment:
Use `"""` always.

Reply:
Oops, but I copied this from proseco which at some point want a fix then
https://github.com/sot/proseco/commit/f7a5a383757f4c3eee50c8fb9238d5964d1b57e3

Reply:
Gahh.  I don't know how I could have done that since it really bothers me when I see `'''`.

Reply:
Looked like it was in pyyaks when I was just over there too, so I have a feeling they have a common ancestor (and that could be my fault)...